lib/src/main/java/com/auth0/jwt/JWTVerifier.java

```diff
@@ -230,6 +230,20 @@ private void requireClaim(String name, Object value) {
         }
     }
 
+    private static boolean checkVarargNull(String[] args) {
```

Comment:
Should this be `checkVarArgNull` ?

Maybe a better name might be isNullOrEmpty?

Reply:
Renamed to `isNullOrEmpty`